### PR TITLE
chore (refs DPLAN-11668): Add missing Hooks for E-2-E Test.

### PR DESCRIPTION
**Pull request by ahmad-demos**
### Ticket
https://demoseurope.youtrack.cloud/issue/DPLAN-11668/Hooks-Public-View-Hooks

- Add missing Hooks/Selectors for DpMultistepNav.

### CODE REVIEW

File: src/components/DpMultistepNav/DpMultistepNav.vue [hunk 1]
```diff
@@ -3,6 +3,7 @@
     <button
       v-for="(step, idx) in steps"
       :key="`step_${idx}`"
+      :data-cy="`${dataCy}:${idx}`"
```
Comment (salisdemos) [inline src/components/DpMultistepNav/DpMultistepNav.vue:6]:
```suggestion
      :data-cy="`${dataCy}:${step.label}`"
```

**Review by salisdemos — CHANGES_REQUESTED** [review]

File: src/components/DpMultistepNav/DpMultistepNav.vue (hunk 1 above)
Comment (ahmad-demos) [reply src/components/DpMultistepNav/DpMultistepNav.vue:6]: With this change we have now hooks like: `data-cy="multistepNav:Ihre Stellungnahme"` with space.

Can I make it like `:data-cy="`${dataCy}:${step.label.replace(/\s+/g, '-').toLowerCase()}`"`?
Then Hook `data-cy="multistepNav:ihre-stellungnahme"`

Comment (ahmad-demos) [reply src/components/DpMultistepNav/DpMultistepNav.vue]: https://github.com/demos-europe/demosplan-ui/pull/881/commits/44354c98fe426a948a840edba3aa62a2ca3a6fd2

Comment (salisdemos) [reply src/components/DpMultistepNav/DpMultistepNav.vue]: Ok. Sorry. Thought the label would look more like a key. than we better stick to the idx.

**Review by salisdemos — CHANGES_REQUESTED** [review]

**Review by salisdemos — APPROVED** [review]

**Pull request merged**